project-base/storefront/hooks/productLists/useProductListUuids.ts

```diff
@@ -0,0 +1,31 @@
+import { usePersistStore } from 'store/usePersistStore';
+
+export const useProductListUuids = (): {
+    getAllProductListUuids: () => string[];
+    removeAllProductListUuids: () => void;
+} => {
+    const wishlistUuid = usePersistStore((store) => store.wishlistUuid);
+    const comparisonUuid = usePersistStore((store) => store.comparisonUuid);
+    const updateWishlistUuid = usePersistStore((store) => store.updateWishlistUuid);
+    const updateComparisonUuid = usePersistStore((store) => store.updateComparisonUuid);
+
+    const getAllProductListUuids = () => {
+        const productListsUuids = [];
+
+        if (wishlistUuid) {
+            productListsUuids.push(wishlistUuid);
+        }
+        if (comparisonUuid) {
+            productListsUuids.push(comparisonUuid);
+        }
+
+        return productListsUuids;
+    };
+
+    const removeAllProductListUuids = () => {
+        updateWishlistUuid(null);
+        updateComparisonUuid(null);
+    };
+
+    return { getAllProductListUuids, removeAllProductListUuids };
+};
```

Comment:
Again, logic split here. I know we probably won't add more list in the near future, but imagine we have 10 types of the list, how many stores/ifs will be here? 🙈

In case of one ProductListUuids store we can skip all this logic.

Reply:
This was completely removed 👍